tests/Console/ConfigurationResolverTest.php

```diff
@@ -868,6 +868,22 @@ public function testResolveRulesWithOption()
         );
     }
 
+    public function testResolveRulesWithUnknownRules()
+    {
+        $this->setExpectedException(
+            'PhpCsFixer\ConfigurationException\InvalidConfigurationException',
+            'The rules contain unknown fixers (foo, bar).'
+        );
+
+        $resolver = new ConfigurationResolver(
+            $this->config,
+            array('rules' => 'foo,-bar'),
```

Comment:
please add existing rule here as well, to be sure tat ex msg is not containing everything